testbed/core/models.py

```diff
@@ -1,38 +1,64 @@
 import logging
 from django.db import models
 from django.contrib.auth.models import User
+from django.contrib.auth import get_user_model
 from django.core.exceptions import ValidationError
 from datetime import timezone
 
 logger = logging.getLogger(__name__)
 
+class ActorManager(models.Manager):
+    # Create both source and destination actors for a user
+    def create_actors_for_user(self, user):
+        source = self.create(
+            user=user,
+            role=Actor.ROLE_SOURCE,
+        )
+        destination = self.create(
+            user=user,
+            role=Actor.ROLE_DESTINATION,
+        )
 
-def validate_username(value):
-    if " " in value or not value.isalnum():
-        raise ValidationError("Username must be alphanumeric and contain no spaces.")
-
-    if len(value) < 3:
-        raise ValidationError("Username must be at least 3 characters.")
+        return source, destination
 
 
 class Actor(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="actor")
-    username = models.CharField(
```

Comment:
I don't think we should delete this - Actor needs a different username than the user because they're separate